templates/page/page.tpl.php

```diff
@@ -132,6 +138,18 @@
   <nav id="ubc7-unit-menu" role="navigation" class="navbar expand">
     <div class="navbar-inner expand">
       <?php print $fluidcontainerstart; ?>
+        <?php if ($drawer_enabled): ?>
+          <!-- UBC Unit Drawer Menu Button -->
+          <button class="hamburger visible-md visible-lg drawer-toggle--primary" id="drawer-button">
+            <div>Menu</div>
```

Comment:
Span tag would be better here and the text could be nice to have it configurable with that as the default in the appearance settings.